packages/core/src/feed.ts

```diff
@@ -1,20 +1,53 @@
-import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitType, LogEntry, StreamMetadata, StreamState } from '@ceramicnetwork/common'
 import { Subject, type Observable } from 'rxjs'
 import { CommitID } from '@ceramicnetwork/streamid'
 import { StreamUtils } from '@ceramicnetwork/common'
 
+/**
+ * Describes all event types
+ */
+export enum EventType {
```

Comment:
I believe the standard names for these from the protocol spec now are Init Events, Data Events, and Time Events.  So presumably that means this enum should have INIT, DATA, and TIME values.

Tagging @oed to confirm.